Turn grc (now ggrc) and fits2pdf scripts into entry points

pllim opened the pull request:
Fixes #303. Also see #200.

For `grc`:

* Moved it to `misc` sub-package as `grc.py`.
* Renamed executable to `ggrc`, which is now an entry point.
* PEP8 fixes.

For `fits2pdf.py` (UPDATED):

* Moved it to `examples/gtk` sub-directory.
* Added a proper option parser.
* PEP8 fixes.

@ejeschke , please give this a spin and merge if satisfactory. c/c @olebole .

Examples (UPDATED):

```
$ which ggrc
~/anaconda/bin/ggrc

$ ggrc --help
Usage: ggrc [options] cmd [arg] ...

Options:
  --version    show program's version number and exit
  -h, --help   show this help message and exit
  --debug      Enter the pdb debugger on main()
  --host=HOST  Connect to server at HOST
  --port=PORT  Connect to server at PORT
  --profile    Run the profiler on main()

$ ggrc --version
2.5.20160603104100
```

ejeschke commented:
@pllim, I think `fits2pdf.py` was just an example program.  If anything, I think we can just move it to `.../ginga/examples/gtk`.  Otherwise the PR looks good to me.

pllim commented:
@ejeschke , it is done. I also removed the `fits2pdf` entry point I added early, as `examples` is not a real sub-package. So, now Ginga only installs `ginga` and `ggrc` executables.

ejeschke commented:
Thanks, @pllim!
